initdb.py

```diff
@@ -0,0 +1,140 @@
+# Initializes a MySQL Database
+# 1) connect to server
+# 2) create database with cursor
+# 3) connect to database
+# 4) create tables with cursor
+# 5) commit transactoins
+# 6) exit
+
+
+
+
+import sys
+import mysql.connector
+from mysql.connector import errorcode
+
+# Define constants
+DB_NAME = "starbucksdb"
+
+TABLES = {}
+# Create table statments
+# replace with own schema
+
+# https://www.kaggle.com/starbucks/store-locations
```

Comment:
Just leaving this comment here to let others know as well, with my task, the datasets will actually be moved into our repo